specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-05-01/containerregistry.json

```diff
@@ -1617,17 +1620,18 @@
       "properties": {
         "days": {
           "format": "int32",
-          "description": "The number of days to retain manifest before it expires.",
+          "description": "The number of days to retain an untagged manifest after which it gets purged.",
+          "default": 7.0,
           "type": "integer"
         },
-        "lastUpdatedTime": {
```

Comment:
this is breaking change: remove property and add property into stable version. pls add new service tag.

Reply:
Removed, not needed currently. Will have this in a future api version.